opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -121,39 +154,122 @@ namespace Opm {
         // calculate the efficiency factors for each well
         calculateEfficiencyFactors();
 
+        const Grid& grid = ebosSimulator_.vanguard().grid();
```

Comment:
Please move this one in the `if (PolymerModule::hasPlyshlog()) {` , because it is only used there.